lib/web/mage/backend/suggest.js

```diff
@@ -245,6 +245,20 @@
 
                         case keyCode.ENTER:
                         case keyCode.NUMPAD_ENTER:
+                            suggestList = event.currentTarget.parentNode.getElementsByTagName('ul')[0];
+                            hasSelectedItems = suggestList.getElementsByClassName('_active').length >= 0;
+
+                            if (hasSelectedItems) {
+                                selectedItem = $(suggestList.getElementsByClassName('_active')[0]);
+                                /* eslint-disable max-depth */
```

Comment:
We do not encourage suppressing warnings from linters. Please consider refactoring this part accordingly.

Reply:
@ishakhsuvarov Yep, I will move this part into a dedicated function I guess.